src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -939,6 +940,13 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         }
     }
 
+    winrt::hstring SaveTaskArgs::GenerateName() const
+    {
+        return winrt::hstring{
+            fmt::format(L"Save Task commandline:{}, name: {}, keyChord {}", Commandline(), Name(), KeyChord())

```

Comment:
nit: we may want to omit the arg from the generated name, if the value of the arg is just empty. Like, in your gif, it always shows "keychord ," even when the keys weren't provided yet

Reply:
Updated, let me know if that looks ok.

For the toast, I have it showing None when the keyChord is null,  is it better to hide the TextBlocks from the toast's content?

Right now if the same name is used I think it will overwrite the existing action,  does it make sense to have a --force arg, and fail if there is a existing action for the name and no force arg?